src/backend/task/pg_cron.c

```diff
@@ -298,6 +300,25 @@ bgw_generate_returned_message(StringInfoData *display_msg, ErrorData edata)
 		appendStringInfo(display_msg, "\nCONTEXT: %s", edata.context);
 }
 
+void
+assign_task_timezone(const char *newval, void *extra)
+{
+	task_timezone_tz = *((pg_tz **) extra);
```

Comment:
suppose extra is reset, task_timezone_tz maybe wild pointer.
```c++
static pg_tz task_timezone_tz;
```
that's better ?

Reply:
okay